ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -114,6 +152,14 @@ export interface Data extends RaidbossData {
   expectedFirstHeadmarker?: string;
   isDoorBoss: boolean;
   phase?: 'superchain1' | 'palladion' | 'superchain2a' | 'superchain2b';
+  engravementCounter: number;
+  engravement1Towers: string[];
+  engravement2MyLabel?: EngravementLabel;
+  engravement3TowerType?: 'lightTower' | 'darkTower';
+  engravement3TowerPlayers: string[];
+  engravement3TetherPlayers: { [name: string]: AnthroposTether };
+  engravement3DarkTethersSide?: 'east' | 'west';
+  engravement3LightTethersSide?: 'east' | 'west';
```

Comment:
Are these redundant? Do we only need one of these variables?

Reply:
It looks like lightTower/darkTower are applied ~1.1s before the adds begin casting, so I think we could just collect the TetherSide that corresponds to the same color as the tower debuff, and use that to resolve the Towers Drop Location.

Reply:
Mostly I was just saying that the dark tethers are one side and the light tethers therefore must be on the opposite side.  So, you only need to track one of them to know where they both are.

Reply:
That is...also fair.  I ended up doing it this way and cut some code in the process, so 🤷‍♂️